lib/WUI/netdev.c

```diff
@@ -184,7 +200,7 @@ uint32_t netdev_init() {
 }
 
 uint32_t netdev_init_esp() {
-    esp_hard_reset_device();
+    // esp_hard_reset_device();
```

Comment:
Any reason why we are not resetting the device here, to make sure it starts in known state?

Reply:
Leaving commented out code behind was a mistake. Integrated upstream changes in LwESP implement hard reset as part of init sequence. We just provide a function pointer to do it.